### ISSUE TITLE
[BUG] - Argo Workflows tab shows up in JupyterHub when Argo Workflows is disabled

### ISSUE THREAD

**Issue by Adam-D-Lewis** [nebari-dev/nebari#1669]
### Describe the bug

Argo Workflows tab shows up in JupyterHub when Argo Workflows is disabled

```yaml
# nebari-config.yaml
...
argo_workflows:
  enabled: false
```
but you still see a tab for it in Jupyterhub
![image](https://user-images.githubusercontent.com/23342526/222299706-0456d2a9-46f7-42e9-92b7-306b02bdae4c.png)

### Expected behavior

No tab when argo workflows is disabled or a page saying Argo Workflows is disabled.  Contact your admin to enable it.

### OS and architecture in which you are running Nebari

N/A

### How to Reproduce the problem?

see above

### Compute environment

None

**Comment by marcelovilla** [comment]
We probably need to take this into account in the new Nebari menu item; see #2182.

**Comment by dharhas** [comment]
I think the new jhub-apps ui, lets you control this more. i.e many of these are optional services and we need a way for the menu and the jhub-app ui to pull the list of services from the config file.

cc: @jbouder

**Comment by jbouder** [comment]
Correct, currently Argo Workflows would only display in the new UI if it was added to the jupyterhub services config.

**Comment by marcelovilla** [comment]
I think we could adapt https://github.com/nebari-dev/nebari/pull/2249 to achieve that
